charts/tfy-jspolicy-configs/templates/replace-image-registry.yaml

```diff
@@ -95,13 +102,14 @@ spec:
       let newImageUri = imageUri;
       let registryReplacement = registryReplacementMap[registry];
       if (!registryReplacement) {
+        if (!registryReplacementMap['*']) {
+          return imageUri;
+        }
         registryReplacement = registryReplacementMap['*'];
-      }
-      if (imageUri.startsWith(registryReplacement)) {
-        return imageUri
       } else {
```

Comment:
What is this else condition for?

Reply:
Else condition is for the case where registry is in the map. First if condition is when registry to be replaced is not in map.

Reply:
But its already assigned the same value in line #96. So this condition shouldn't be needed